core/templates/pages/learner-dashboard-page/new-learner-dashboard-page/home-tab.component.html

```diff
@@ -0,0 +1,238 @@
+<div class="greeting">
+  Good {{getTimeOfDay()}}, {{username}}!
+</div>
+<div class="continue-where-you-left-off" *ngIf="isNonemptyObject(untrackedTopics) || currentGoals.length !== 0">
+  <p class="continue-where-you-left-off-heading" [innerHTML]="'I18N_LEARNER_DASHBOARD_CONTINUE_WHERE_YOU_LEFT_OFF_SECTION' | translate">
+  </p>
+  <div class="empty-continue-where-you-left-off-section" *ngIf="currentGoals.length === 0">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_CONTINUE_WHERE_YOU_LEFT_OFF_SECTION' | translate" class="empty-continue-where-you-left-off-section-text">
+    </p>
+    <a id="setting-a-goal-link" (click)="changeActiveSection()">
+      setting a goal!
+    </a>
+  </div>
+  <div class="continue-where-you-left-off-scroll">
+    <div class="continue-where-you-left-off-section" *ngIf="currentGoals.length !== 0" [ngStyle]="checkMobileView() && {'width': width + 'px'}">
+      <div *ngFor="let topicSummaryTile of currentGoals" class="topic-summary">
+        <div *ngFor="let storySummaryTile of topicSummaryTile.canonicalStorySummaryDicts; let i = index"
+             class="story-summary-tile">
+          <oppia-learner-story-summary-tile [storySummary]="storySummaryTile" [displayArea]="'homeTab'"
+            [topicName]="topicSummaryTile.name">
+          </oppia-learner-story-summary-tile>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="suggested-for-you">
+  <p class="suggested-for-you-heading"
+    [innerHTML]="'I18N_LEARNER_DASHBOARD_SUGGESTED_FOR_YOU_SECTION' | translate">
+  </p>
+  <div class="empty-suggested-for-you-section" *ngIf="!isNonemptyObject(untrackedTopics)">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUGGESTED_FOR_YOU_SECTION' | translate">
+    </p>
+  </div>
+  <div class="suggested-for-you-section" *ngIf="isNonemptyObject(untrackedTopics)">
+    <div *ngFor="let item of untrackedTopics | keyvalue" class="untracked-topics">
+      <div *ngIf="item.value.length !== 0" class="classroom-name">
+        {{item.key}}
+      </div>
+      <div class="topic-summary-tile-scroll">
+        <div class="topic-summary-tile-section" [ngStyle]="checkMobileView() && {'width': getWidth(item.value.length) + 'px'}">
+          <div *ngFor="let topicSummaryTile of item.value.slice(0,3)" class="topic-summary-tile">
+            <oppia-learner-topic-summary-tile [topicSummary]="topicSummaryTile">
+            </oppia-learner-topic-summary-tile>
+          </div>
+          <a [href]="getClassroomLink(item.key)" target="{{ openInNewWindow ? '_blank' : '_self' }}"
+             class="oppia-classroom-link">
+            <div class="classroom-card">
+              <p class="classroom-text">
+                View all {{item.key}} topics
+              </p>
+            </div>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<style>
+  .greeting {
```

Comment:
Can all these css classes be wrapped in a parent div class?

Reply:
This div does not come under any other div on this page. I have renamed it to `oppia-greeting`. Also, all the other CSS classes are wrapped in a parent div. PTAL!